styles_src/default.scss

```diff
@@ -11,7 +11,7 @@ $party-margin: 2px;
 
   /* subtitle */
   &-info {
```

Comment:
can you remove this and use `.s-q-item-subtitle` from https://github.com/nzzdev/sophie-q/blob/master/scss/title.scss#L29 instead?